test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/HistoryTests.cs

```diff
@@ -127,20 +128,22 @@ public async Task GivenAValueForSince_WhenGettingSystemHistory_TheServerShouldRe
 
             var updatedResource = await CreateResourceWithTag(_createdResource.Resource, tag);
 
-            using FhirResponse<Bundle> readResponse = await _client.SearchAsync($"_history?_tag={tag}&_since={sinceUriString}");
+            using FhirResponse<Bundle> readResponse = await _client.SearchAsync($"_history?_tag={tag}&_since={sinceUriString}&_count={count}");
 
-            AssertCount(1, readResponse.Resource.Entry);
+            var readResponseWithMatchingTag = readResponse.Resource.Entry.Where(e => e.Resource.Meta.Tag.Where(t => t.Code == tag).Any()).ToList();
```

Comment:
Nit: ReSharper is telling me this can be simplified slightly

`var readResponseWithMatchingTag = readResponse.Resource.Entry.Where(e => e.Resource.Meta.Tag.Any(t => t.Code == tag)).ToList();`